esp/bindings/http/platform/httpbinding.cpp

```diff
@@ -727,6 +727,37 @@ void EspHttpBinding::addToESPCache(CHttpRequest* request, CHttpResponse* respons
         ESPLOG(LogMax, "Failed to add ESP Cache for %s.", method);
 }
 
+void EspHttpBinding::clearCacheByGroupID(const char *ids)
+{
+    if (isEmptyString(ids))
```

Comment:
How much overhead is this imposing on systems that do not have memcached installed?

Reply:
I may add a quick check to avoid possible overhead.